apps/frontend/src/components/CharityCard.tsx

```diff
@@ -0,0 +1,122 @@
+import { Card, CardContent, Typography, Grid, Avatar } from "@mui/material";
+import { useStyles } from "../styles/pageStyles";
+import { colors } from "../styles/colors";
+
+interface Organization {
+  name: string;
+  logoUrl: string;
+  donated: number;
+  receiving: boolean;
+}
+
+interface CharityCardProps {
+  id: string;
+  name: string;
+  description: string;
+  startDate: string;
+  endDate: string;
+  imageUrl: string;
+  organizations: Organization[];
+  funds: number;
+  listingsCount: number;
+}
+
+function formatDate(dateString: string) {
```

Comment:
JS probably has a date .toString() function to format dates, if the date comes in as a parsable date object. Would need a bit of research

Reply:
<img width="827" alt="Screenshot 2024-07-13 at 1 43 55 PM" src="https://github.com/user-attachments/assets/9323b164-5076-48df-a077-dfbf748992df">

I tried for quite some time this morning to use a library, but I keep getting the above error when I try and use a library.

I did run `npm install date-fns`